packages/components/src/query-controls/index.js

```diff
@@ -37,6 +37,14 @@ export default function QueryControls( {
 	onOrderChange,
 	onOrderByChange,
 } ) {
+	const suggestions = categoriesList.reduce(
```

Comment:
Let's call this `categorySuggestions` because if we add other flat taxonomies they'll also have suggestions.